src/less/split-button/base/split-button.less

```diff
@@ -0,0 +1,18 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.split-button button.btn--split-start {
+    border-radius: 24px 0 0 24px;
+}
+
+.split-button button.btn--split-end {
+    border-left-color: currentColor;
+    border-radius: 0 24px 24px 0;
+    margin-left: -5px;
+    min-width: 40px;
+    padding-left: 8px;
+    padding-right: 8px;
+}
+
+.split-button button.btn--tertiary.btn--split-end {
+    .border-left-color-token(tertiary-border-left-color, color-separator);
+}
```

Comment:
Okay, so I didn't notice this, sorry. 

Lets put split button just to import button and menu. I think it should be a no-op file.
This code should live in button and then the top level file should import `menu-button` and `button`

Reply:
Agreed.

Reply:
okay let me know what you think with these latest changes